js-core/lesson1.md

```diff
@@ -4,198 +4,428 @@
+# Getting Started
 
-**TODO**
+1. Download and install the latest Current version of NodeJS - from https://nodejs.org/en/download/current/
 
-### Comparisons
+To test that it was installed and running properly, go to your terminal and run the command:
+`node -v`
+You should get the node version printed on your terminal, for example, `v7.3.0`
 
-**TODO**
 
-- Introduce `==` and `!=`, `<` and `>`
-- Introduce the concept of truthiness and falsiness for non-boolean types
-- Introduce `===` and `!==`
+2. Fork the repositroy - https://github.com/Code-Your-Future/JS-Core-1-Exercises and **Clone** it locally.
 
-### If/Else
+3. Once you have the repo locally, go to the folder on your terminal and run `node main.js`
 
-**TODO**
+You should get **Hello World** printed on your Terminal.
 
-### Switch
 
-**TODO**
+Fork the repositroy - https://github.com/Code-Your-Future/JS-Core-1-Exercises and **Clone** it locally.
 
-### The ternary operator
+# Hello World
 
-**TODO**
+It is programming tradition that the first thing you do in any language is make it say "Hello world!". This is the first thing we'll do in JavaScript, using something called a `console.log()`.
 
-## Arrays
+## `console.log()`
 
-If you want to store a bunch of things, identifiers like `name1`, `name2`, `name3` and so on quickly become tiresome. **Arrays** to the rescue!
+When we ran the command `node main.js` - that did **log** to the **console** printing the words **Hello World** to the terminal. 
 
-Imagine an array like a list of variables, but instead of each variable having its own name, only the array has a name. So how do you access a variable inside, then? How do you read and write? Using an *index*!
+Let's open the file **main.js** and investigate its contents
 
-An index is just a whole number, such as `0`, `1`, `2` and so forth. As in many areas of computing, indices in JavaScript arrays start with `0` --- so `0` is the index of the first element of an Array, `1` is the index of the second element, and so forth.
+> **Exercises**
+> - What do you notice about the code? Anything you don't understand? Notice the semicolon, the dot, the parantheses - these are all elements of JavaScript **Syntax**
+>
+> - Try to `console.log()` something different, for example, `Hello World. I just started learning JavaScript!`. (run `node main.js` after each change and look at the output in the console)
+> - Try to `console.log()` several things at once.
+> - What happens when you get rid of the quote marks?
+> - What happens when you `console.log()` just a number without quotes?
+
+## Comments
+
+You can put comments to help you and others understand your code - use `//` to add a comment (or if it is a multi-line comment, then use `/*` and `*/` around your comment)
+```js
+// This is a comment for myself and other developers to understand the next line of code
+console.log(typeof 'Hello world!');
+
+console.log(typeof 'Hello World'); // comments can also be in the same line like this
```

Comment:
Hmm, was it intentional to add `typeof` here? Might confuse people, as the point of this topic is just to introduce comments?